### PR TITLE
ID Localization for Limit Ranges Concept.

### PR DESCRIPTION
**Pull request by ariscahyadi**
This PR for addressing translation project as mentioned in #20111

### CODE REVIEW

File: content/id/docs/concepts/policy/limit-range.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: Batas Rentang (Limit Range)
```
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/concepts/policy/limit-range.md:2]: Done!

File: content/id/docs/concepts/policy/limit-range.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+- Menerapkan penggunaan sumber daya komputasi minimum dan maksimum untuk setiap Pod atau Container dalam _namespace_.
+- Menerapkan permintaan tempat penyimpanan minimum dan maksimum untuk setiap PersistentVolumeClaim dalam _namespace_.
+- Menerapkan rasio antara permintaan dan batas untuk sumber daya dalam _namespace_.
+- Menetapkan permintaan/batas bawaan untuk menghitung sumber daya dalam _namespace_ dan memasukkannya secara otomatis ke Container pada _runtime_.
+
+## Mengaktifkan LimitRange
+
+Dukungan LimitRange diaktifkan secara bawaan untuk banyak distribusi Kubernetes. Hal ini
+diaktifkan ketika tanda `--enable-admission-plugins =` pada apiserver memiliki admission controller `LimitRanger` sebagai
```
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/concepts/policy/limit-range.md:28]: Thank @sftim for the reminder. Sorry for that :)

**Comment by ariscahyadi** [comment]
> We've [stopped using capture shortcodes](https://github.com/kubernetes/website/pull/21359)
> 
> Please revise this PR to match the recently-updated English language original.

Thanks for the correction, I updated with <-- overview --> and <-- body -->

**Comment by ariscahyadi** [comment]
> > We've [stopped using capture shortcodes](https://github.com/kubernetes/website/pull/21359)
> > Please revise this PR to match the recently-updated English language original.
> 
> Thanks for the correction, I updated with <-- overview --> and <-- body -->

@sftim the deployment is failed due to this heading `## {{% heading "whatsnext" %}}`, which is the same as the English version.

**Comment by sftim** [comment]
@ariscahyadi if you rebase this pull request against the _master_ branch it should preview OK.

**Comment by ariscahyadi** [comment]
> @ariscahyadi if you rebase this pull request against the _master_ branch it should preview OK.

Thanks @sftim it is work!

File: content/id/docs/concepts/policy/limit-range.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+---
+title: Batas Rentang (Limit Range)
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:2]: LimitRange is an API object, let's just use LimitRange without translation as title?
we can add "Batas Rentang" in the beginning of paragraph to add meaning to it

File: content/id/docs/concepts/policy/limit-range.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+content_template: templates/concept
+weight: 10
+---
+
+<!-- overview -->
+
+Secara bawaan, Container berjalan dengan [sumber daya komputasi](id/docs/user-guide/compute-resources) tanpa batas pada klaster Kubernetes.
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:9]: missing `/` for the link
```suggestion
Secara bawaan, Container berjalan dengan [sumber daya komputasi](/id/docs/user-guide/compute-resources) tanpa batas pada klaster Kubernetes.
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+Dengan kuota sumber daya, administrator klaster dapat membatasi konsumsi dan pembuatan sumber daya berbasis {{<glossary_tooltip text="namespace" term_id="namespace">}}.
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:10]: ResourceQuota is a Kubernetes api object, need to add here as well

Namespace should start with uppercase
```suggestion
Dengan ResourceQuota (kuota sumber daya), administrator klaster dapat membatasi konsumsi dan pembuatan sumber daya berbasis {{<glossary_tooltip text="Namespace" term_id="namespace">}}.
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+Di dalam _namespace_, Pod atau Container dapat mengkonsumsi CPU dan memori sesuai dengan yang ditentukan oleh kuota sumber daya _namespace_. Ada kekhawatiran bahwa satu Pod atau Container dapat memonopoli semua sumber daya yang tersedia. LimitRange adalah kebijakan untuk membatasi alokasi sumber daya (bagi Pod atau Container) pada _namespace_.
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:11]: Namespace is kubernetes object
```suggestion
Di dalam Namespace, Pod atau Container dapat mengkonsumsi CPU dan memori sesuai dengan yang ditentukan oleh ResourceQuota pada Namespace tersebut. Ada kekhawatiran bahwa satu Pod atau Container dapat memonopoli semua sumber daya yang tersedia. LimitRange adalah kebijakan untuk membatasi alokasi sumber daya (bagi Pod atau Container) pada Namespace.
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+
+<!-- body -->
+
+LimitRange memberikan batasan yang dapat:
+
+- Menerapkan penggunaan sumber daya komputasi minimum dan maksimum untuk setiap Pod atau Container dalam _namespace_.
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:18]:
```suggestion
- Menerapkan penggunaan sumber daya komputasi minimum dan maksimum untuk setiap Pod atau Container dalam Namespace.
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+- Menerapkan permintaan tempat penyimpanan minimum dan maksimum untuk setiap PersistentVolumeClaim dalam _namespace_.
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:19]:
```suggestion
- Menerapkan permintaan tempat penyimpanan minimum dan maksimum untuk setiap PersistentVolumeClaim dalam Namespace.
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+- Menerapkan rasio antara permintaan dan batas untuk sumber daya dalam _namespace_.
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:20]:
```suggestion
- Menerapkan rasio antara permintaan dan batas untuk sumber daya dalam Namespace.
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+- Menetapkan permintaan/batas bawaan untuk menghitung sumber daya dalam _namespace_ dan memasukkannya secara otomatis ke Container pada _runtime_.
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:21]:
```suggestion
- Menetapkan permintaan/batas bawaan untuk menghitung sumber daya dalam Namespace dan memasukkannya secara otomatis ke Container pada _runtime_.
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+
+## Mengaktifkan LimitRange
+
+Dukungan LimitRange diaktifkan secara bawaan untuk banyak distribusi Kubernetes. Hal ini
+diaktifkan ketika tanda `--enable-admission-plugins=` pada apiserver memiliki admission controller `LimitRanger` sebagai
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:26]: nit
```suggestion
diaktifkan ketika tanda `--enable-admission-plugins=` pada apiserver memiliki _admission controller_ `LimitRanger` sebagai
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+salah satu argumennya.
+
+LimitRange diberlakukan pada _namespace_ tertentu ketika ada sebuah objek LimitRange pada _namespace_ tersebut.
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:29]:
```suggestion
LimitRange diberlakukan pada Namespace tertentu ketika ada sebuah objek LimitRange pada Namespace tersebut.
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+
+Nama dari objek LimitRange harus merupakan sebuah [nama subdomain DNS](id/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:31]: missing `/` and incorrect anchor link
```suggestion
Nama dari objek LimitRange harus merupakan sebuah [nama subdomain DNS](/id/docs/concepts/overview/working-with-objects/names/#nama).
```

Comment (girikuncoro) [reply content/id/docs/concepts/policy/limit-range.md:31]: we might haven't translated the new subsections in the DNS page

File: content/id/docs/concepts/policy/limit-range.md [hunk 14]
```diff
@@ -0,0 +1,65 @@
...
+
+### Gambaran Umum LimitRange
+
+- Administrator membuat sebuah LimitRange dalam sebuah _namespace_.
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:35]:
```suggestion
- Administrator membuat sebuah LimitRange dalam sebuah Namespace.
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 15]
```diff
@@ -0,0 +1,65 @@
...
+- Pengguna membuat sumber daya seperti Pod, Container, dan PersistentVolumeClaims pada namespace.
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:36]: nit: use singular form
```suggestion
- Pengguna membuat sumber daya seperti Pod, Container, dan PersistentVolumeClaim pada Namespace.
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 16]
```diff
@@ -0,0 +1,65 @@
...
+- _Admission controller_ `LimitRanger` memberlakukan bawaan dan batas untuk semua Pod dan Container yang tidak menetapkan persyaratan sumber daya komputasi dan melacak penggunaannya untuk memastikan agar tidak melebihi minimum, maksimum dan rasio sumber daya yang ditentukan dalam LimitRange yang ada pada _namespace_.
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:37]:
```suggestion
- _Admission controller_ `LimitRanger` memberlakukan bawaan dan batas untuk semua Pod dan Container yang tidak menetapkan persyaratan sumber daya komputasi dan melacak penggunaannya untuk memastikan agar tidak melebihi minimum, maksimum dan rasio sumber daya yang ditentukan dalam LimitRange yang ada pada Namespace.
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+- Apabila permintaan membuat atau memperbarui sumber daya (Pod, Container, PersistentVolumeClaim) yang melanggar batasan LimitRange, maka permintaan ke server API akan gagal dengan kode status HTTP `403 FORBIDDEN` dan sebuah pesan yang menjelaskan batasan yang telah dilanggar.
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:38]: server API or API server? Make it consistent, both are actually valid
```suggestion
- Apabila permintaan membuat atau memperbarui sumber daya (Pod, Container, PersistentVolumeClaim) yang melanggar batasan LimitRange, maka permintaan ke API server akan gagal dengan kode status HTTP `403 FORBIDDEN` dan sebuah pesan yang menjelaskan batasan yang telah dilanggar.
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+- Apabila LimitRange diaktifkan pada _namespace_ untuk menghitung sumber daya seperti `cpu` dan `memory`, pengguna harus menentukan permintaan atau batasan untuk nilai-nilai itu. Jika tidak, sistem dapat menolak pembuatan Pod.
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:39]:
```suggestion
- Apabila LimitRange diaktifkan pada Namespace untuk menghitung sumber daya seperti `cpu` dan `memory`, pengguna harus menentukan permintaan atau batasan untuk nilai-nilai itu. Jika tidak, sistem dapat menolak pembuatan Pod.
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 19]
```diff
@@ -0,0 +1,65 @@
...
+- Pelanggaran terhadap LimitRange hanya terjadi pada tahap penerimaan Pod, bukan pada saat Pod sedang berjalan.
+
+Contoh dari kebijakan yang dapat dibuat dengan menggunakan LimitRange yaitu:
+
+- Dalam klaster dua Node dengan kapasitas 8 GiB RAM dan 16 _core_, batasan Pod dalam _namespace_ meminta 100m untuk CPU dengan batas maksimum 500m untuk CPU dan minta 200Mi untuk Memori dengan batas maksimum 600Mi untuk Memori.
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:44]:
```suggestion
- Dalam klaster dua Node dengan kapasitas 8 GiB RAM dan 16 _core_, batasan Pod dalam Namespace meminta 100m untuk CPU dengan batas maksimum 500m untuk CPU dan minta 200Mi untuk Memori dengan batas maksimum 600Mi untuk Memori.
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 20]
```diff
@@ -0,0 +1,65 @@
...
+- Tetapkan batas bawaan dan permintaan pada 150m untuk CPU dan permintaan standar memori pada 300Mi untuk Container yang dimulai tanpa cpu dan permintaan memori dalam spesifikasi mereka.
+
+Dalam kasus di mana batas total _namespace_ kurang dari jumlah batas Pod/Container,
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:47]:
```suggestion
Dalam kasus di mana batas total Namespace kurang dari jumlah batas Pod/Container,
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 21]
```diff
@@ -0,0 +1,65 @@
...
+mungkin akan ada perebutan untuk sumber daya. Dalam hal ini, maka Container atau Pod tidak akan dibuat.
+
+Baik perebutan maupun perubahan pada LimitRange tidak akan memengaruhi sumber daya yang sudah dibuat.
+
+
+## {{% heading "whatsnext" %}}
+
+Silahkan merujuk pada [Dokumen desain LimitRanger](https://git.k8s.io/community/contributors/design-proposals/resource-management/admission_control_limit_range.md) untuk informasi lebih lanjut.
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:55]: nit: desain can also be "perancangan"
```suggestion
Silahkan merujuk pada [Dokumen perancangan LimitRanger](https://git.k8s.io/community/contributors/design-proposals/resource-management/admission_control_limit_range.md) untuk informasi lebih lanjut.
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 22]
```diff
@@ -0,0 +1,65 @@
...
+
+Untuk contoh tentang penggunaan batas, lihatlah:
+
+- [Bagaimana cara mengonfigurasi batasan CPU minimum dan maksimum untuk setiap _namespace_](/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace/).
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:59]:
```suggestion
- [Bagaimana cara mengonfigurasi batasan CPU minimum dan maksimum untuk setiap Namespace](/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace/).
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 23]
```diff
@@ -0,0 +1,65 @@
...
+- [Bagaimana cara mengonfigurasi batasan memori minimum dan maksimum untuk setiap _namespace_](/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace/).
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:60]:
```suggestion
- [Bagaimana cara mengonfigurasi batasan memori minimum dan maksimum untuk setiap Namespace](/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace/).
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 24]
```diff
@@ -0,0 +1,65 @@
...
+- [Bagaimana cara mengonfigurasi permintaan dan batas bawaan CPU untuk setiap _namespace_](/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace/).
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:61]:
```suggestion
- [Bagaimana cara mengonfigurasi permintaan dan batas bawaan CPU untuk setiap Namespace](/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace/).
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 25]
```diff
@@ -0,0 +1,65 @@
...
+- [Bagaimana cara mengonfigurasi Permintaan dan Batas bawaan memori untuk setiap _namespace_](/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/).
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:62]:
```suggestion
- [Bagaimana cara mengonfigurasi permintaan dan batas bawaan memori untuk setiap Namespace](/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/).
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 26]
```diff
@@ -0,0 +1,65 @@
...
+- [Bagaimana cara mengonfigurasi konsumsi tempat penyimpanan minimum dan maksimum untuk setiap _namespace_](/docs/tasks/administer-cluster/limit-storage-consumption/#limitrange-to-limit-requests-for-storage).
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:63]:
```suggestion
- [Bagaimana cara mengonfigurasi konsumsi tempat penyimpanan minimum dan maksimum untuk setiap Namespace](/docs/tasks/administer-cluster/limit-storage-consumption/#limitrange-to-limit-requests-for-storage).
```

File: content/id/docs/concepts/policy/limit-range.md [hunk 27]
```diff
@@ -0,0 +1,65 @@
...
+- [Contoh terperinci tentang mengonfigurasi kuota untuk setiap _namespace_](/docs/tasks/administer-cluster/quota-memory-cpu-namespace/).
```
Comment (girikuncoro) [inline content/id/docs/concepts/policy/limit-range.md:64]:
```suggestion
- [Contoh terperinci tentang mengonfigurasi kuota untuk setiap Namespace](/docs/tasks/administer-cluster/quota-memory-cpu-namespace/).
```

File: content/id/docs/concepts/policy/limit-range.md (hunk 3 above)
Comment (ariscahyadi) [reply content/id/docs/concepts/policy/limit-range.md:2]: Fixed.

File: content/id/docs/concepts/policy/limit-range.md (hunk 4 above)
Comment (ariscahyadi) [reply content/id/docs/concepts/policy/limit-range.md:9]: Fixed.

File: content/id/docs/concepts/policy/limit-range.md (hunk 5 above)
Comment (ariscahyadi) [reply content/id/docs/concepts/policy/limit-range.md:10]: Thanks for the correction!

File: content/id/docs/concepts/policy/limit-range.md (hunk 6 above)
Comment (ariscahyadi) [reply content/id/docs/concepts/policy/limit-range.md:11]: Done!

File: content/id/docs/concepts/policy/limit-range.md (hunk 11 above)
Comment (ariscahyadi) [reply content/id/docs/concepts/policy/limit-range.md:26]: Done!

File: content/id/docs/concepts/policy/limit-range.md (hunk 13 above)
Comment (ariscahyadi) [reply content/id/docs/concepts/policy/limit-range.md:31]: Ok!

File: content/id/docs/concepts/policy/limit-range.md (hunk 15 above)
Comment (ariscahyadi) [reply content/id/docs/concepts/policy/limit-range.md:36]: Done!

File: content/id/docs/concepts/policy/limit-range.md (hunk 17 above)
Comment (ariscahyadi) [reply content/id/docs/concepts/policy/limit-range.md:38]: I prefer `server API`.

File: content/id/docs/concepts/policy/limit-range.md (hunk 21 above)
Comment (ariscahyadi) [reply content/id/docs/concepts/policy/limit-range.md:55]: Done!

File: content/id/docs/concepts/policy/limit-range.md (hunk 25 above)
Comment (ariscahyadi) [reply content/id/docs/concepts/policy/limit-range.md:62]: Done!

**Comment by ariscahyadi** [comment]
> Thanks for localizing this page @ariscahyadi ! Just couple of nits and you are good to go
> 
> Mostly:
> 
> * Namespace is a Kubernetes object https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#namespace-v1-core so must be written in CamelCase format
> * "batasan" (_request_) and "permintaan" (_limit_) is a field in Container spec, let's add 1 explanation in the beginning somewhere before using this term?

You're welcome @girikuncoro! It is also a good opportunity for me.
But, I think these are what you mean "batasan" (limit) and "permintaan" (request) :)

**Comment by girikuncoro** [comment]
looking good, thanks for this @ariscahyadi 

/lgtm
/approve

**Pull request merged**